packages/mask/src/plugins/NFTCard/SNSAdaptor/NFTCardDialogUI.tsx

```diff
@@ -0,0 +1,123 @@
+    // SourceType.LooksRare,
+]
+
+export function NFTCardDialogUI(props: NFTCardDialogUIProps) {
+    const { currentTab, asset, orders, onChangeProvider, provider, events } = props
+    const { classes } = useStyles()
+    const { t: tb } = useBaseI18n()
+    const { setDialog: setSelectProviderDialog } = useRemoteControlledDialog(
+        WalletMessages.events.selectProviderDialogUpdated,
+    )
+    const pluginId = useCurrentWeb3NetworkPluginID()
+    const chainIdList =
+        useActivatedPlugin(PluginId.NFTCard, 'any')?.enableRequirement.web3?.[NetworkPluginID.PLUGIN_EVM]
+            ?.supportedChainIds ?? []
+    console.log(asset, 'ggg')
+    return (
+        <div className={classes.contentWrapper}>
+            {(asset.value && (
+                <div className={classes.contentLayout}>
+                    <div className={classes.mediaBox}>
+                        <NFTBasicInfo
+                            timeline
+                            providers={supportedProvider}
+                            currentProvider={provider}
+                            asset={asset.value}
+                            onChangeProvider={onChangeProvider}
+                        />
+                    </div>
+
+                    <div className={classes.tabWrapper}>
+                        {currentTab === NFTCardDialogTabs.About ? (
+                            <AboutTab orders={orders} asset={asset} />
+                        ) : currentTab === NFTCardDialogTabs.Offers ? (
+                            <OffersTab offers={orders} provider={provider} />
+                        ) : (
+                            <ActivityTab events={events} />
+                        )}
+                    </div>
+                </div>
+            )) ||
+                (asset.loading ? (
+                    <div className={classes.contentWrapper}>
+                        <div className={classes.loadingPlaceholder}>
+                            <LoadingBase />
+                        </div>
+                    </div>
+                ) : (
+                    <div className={classes.contentWrapper}>
+                        <div className={classes.loadingPlaceholder}>
+                            <Typography className={classes.emptyText}>{tb('plugin_furucombo_load_failed')}</Typography>
+                            <Button variant="text" onClick={() => asset.retry()}>
+                                {tb('retry')}
+                            </Button>
+                        </div>
+                    </div>
+                ))}
+
+            <PluginWalletStatusBar className={classes.footer}>
+                <Button
+                    variant="contained"
+                    size="medium"
+                    onClick={() => {
+                        setSelectProviderDialog({
+                            open: true,
+                            supportedNetworkList: chainIdList
+                                ?.map((chainId) => {
```

Comment:
It has been fallback chainIdList to `[]`, it couldn't be nullish
```suggestion
                                .map((chainId) => {
```